### PR TITLE
Improve contracts and guardian node

### PR DESCRIPTION
**Pull request by Lbqds**
* Resolve review comments
* Improve guardian node and bridge UI

### CODE REVIEW

File: alephium/contracts/sequence/unexecuted_sequence.ral [hunk 1]
```diff
@@ -1,11 +1,11 @@
-Contract UndoneSequence(
+Contract UnExecutedSequence(
```
Comment (polarker) [inline alephium/contracts/sequence/unexecuted_sequence.ral:1]: Discussion: it seems `unexecuted` is valid English word, we could use `UnexecutedSequence` for contract name and `unexecutedSequence` for variables.

@h0ngcha0 What's your thoughts? I am not good at English.

File: alephium/contracts/token_bridge/token_bridge.ral [hunk 2]
```diff
@@ -129,26 +129,15 @@ Contract TokenBridge(
...
-        )
-    }
-
     @using(preapprovedAssets = true)
     pub fn createLocalTokenPool(localTokenId: ByteVec, payer: Address, createContractAlphAmount: U256) -> () {
-        assert!(tokenRemaining!(payer, localTokenId) == 1, ErrorCodes.TokenNotExist) // make sure that the token exists
+        let isWrappedAlph = localTokenId == wrappedAlphId
+        let templateContractId =
+            if (isWrappedAlph) tokenBridgeFactory.getWrappedAlphPoolTemplateId()
+            else tokenBridgeFactory.getLocalTokenPoolTemplateId()
```
Comment (polarker) [inline alephium/contracts/token_bridge/token_bridge.ral:40]: [Nit]: move this line just above the following `copyCreateSubContract`, so the variable is created when it's used and review will be slightly easier.

**Review by polarker — APPROVED** [review]
LGTM 👍 Just minor comments.

File: alephium/contracts/sequence/unexecuted_sequence.ral (hunk 1 above)
Comment (h0ngcha0) [reply alephium/contracts/sequence/unexecuted_sequence.ral:1]: Sounds good :+1:

**Pull request merged**